hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```diff
@@ -79,16 +79,18 @@ private ContainerUtils() {
   public static ContainerCommandResponseProto logAndReturnError(
       Logger log, StorageContainerException ex,
       ContainerCommandRequestProto request) {
-    String logInfo = "Operation: {} , Trace ID: {} , Message: {} , " +
-        "Result: {} , StorageContainerException Occurred.";
+    String logInfo = "Operation: {} , Trace ID: {} , Block ID: {} , " +
+        "Message: {} , Result: {} , StorageContainerException Occurred.";
     if (ex.getResult() == CLOSED_CONTAINER_IO ||
         ex.getResult() == CONTAINER_NOT_OPEN) {
       if (log.isDebugEnabled()) {
-        log.debug(logInfo, request.getCmdType(), request.getTraceID(),
+        log.warn(logInfo, request.getCmdType(), request.getTraceID(),
+            request.getGetBlock().getBlockID(),
```

Comment:
I think this should be still logged at DEBUG level if it is under the `isDebugEnabled()` condition.

Reply:
thanks for your comment
I commit new version that if it is under the condition still logged at DEBUG level. 
if it's not under the condition logged at Warn level.